templates/client-auth-n.cfg.xml.erb

```diff
@@ -3,7 +3,7 @@
 <client-auth xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host') %>/repose/client-auth/v1.0">
 <%- if ! @delegating.nil? -%>
   <%- if @delegating %>
-    <delegating/>
+    <delegating<%- if ! @delegating_quality.nil? -%> quality="<%= @delegating_quality %>"<%- end -%>/>
```

Comment:
No you're right. Perhaps they fixed how this was handled or it's only when you use <%= -%>.  Either way the -%> is only really necessary when you want to remove line breaks. http://stackoverflow.com/questions/7996695/what-is-the-difference-between-and-in-erb-in-rails has a good list of what each one might do.